spec/imsc-hrm.html

```diff
@@ -560,7 +562,7 @@ <h2>Paint Regions</h2>
 
       <p class="equation">S(E<sub>n</sub>) = CLEAR(E<sub>n</sub>) + PAINT(E<sub>n</sub> )</p>
 
-      <p>where CLEAR(E<sub>n</sub>) = 0 if n = 0; and CLEAR(E<sub>n</sub>) = 1 otherwise.</p>
+      <p>where CLEAR(E<sub>n</sub>) = 0 if n == 0; and CLEAR(E<sub>n</sub>) == 1 otherwise.</p>
```

Comment:
Looks like the last `==` is supposed to be an assignment operator not an equality comparison, so should be reverted to a single `=`?

Reply:
Fixed at https://github.com/w3c/imsc-hrm/pull/50/commits/82fcc9d52f4268a06fe3a7cca010ac3ec4f1ffc7